src/components/AddToMetamask/index.tsx

```diff
@@ -31,22 +42,25 @@ function AddToMetamaskButton({ token, noBackground, ...otherProps }: { token: To
 
   const { symbol, address, decimals } = token
 
+  function addToken() {
+    registerToken(address, symbol!, decimals)
+  }
+
   return library?.provider?.isMetaMask ? (
-    <MouseoverTooltip text={`Add ${symbol} to Metamask`}>
-      {noBackground ? (
-        <StyledMetamaskImg
-          src={MetamaskIcon}
-          alt={'Metamask logo'}
-          onClick={() => registerToken(address, symbol!, decimals)}
-          {...otherProps}
-        />
-      ) : (
-        <StyledMetamaskButton onClick={() => registerToken(address, symbol!, decimals)} {...otherProps}>
-          <img src={MetamaskIcon} alt={'Metamask logo'} style={{ width: '100%' }} />
-        </StyledMetamaskButton>
-      )}
-    </MouseoverTooltip>
+    textOnly ? (
```

Comment:
Why do you need `textOnly`?  If this is `true` in all call sites, then this can be removed.

Reply:
I added textOnly for this new case of AddToMetaMask, where it only renders as a text link, instead of the fox button. While the fox version is currently in pools page merged in Main already.

If we want to remove that before implementing the CurrencyLogo Wrapper implementation, I can do it in another PR if you agree

Reply:
edit: Removing prop as we decided on not using fox icon